src/Features/PlanningEvent.php

```diff
@@ -720,7 +720,7 @@ public static function showRepetitionForm(string $rrule = "", array $options = [
     {
         $rrule = json_decode($rrule, true) ?? [];
         $defaults = [
-         'freq'       => null,
+         'freq'       => '',
```

Comment:
Not sure about this one.

Call to `Dropdown::showFromArray()` defines `null => __("Never"),` as a possible value and some lines are comparing `$rrule['freq'] == null`. Maybe the line 746 should be changed to `'value' => $rrule['freq'] !== null ? strtolower($rrule['freq']) : null`.